llvm/test/CodeGen/AMDGPU/gfx10-twoaddr-fma.mir

```diff
@@ -1,29 +1,120 @@
 # RUN: llc -mtriple=amdgcn -mcpu=gfx1010 %s -run-pass twoaddressinstruction -verify-machineinstrs -o - | FileCheck --check-prefixes=GFX10 %s
 # RUN: llc -mtriple=amdgcn -mcpu=gfx1010 %s --passes=two-address-instruction -o - | FileCheck --check-prefixes=GFX10 %s
+# RUN: llc -march=amdgcn -mcpu=gfx1010 -run-pass liveintervals -run-pass twoaddressinstruction -verify-machineinstrs %s -o -
```

Comment:
```suggestion
# RUN: llc -mtriple=amdgcn -mcpu=gfx1010 -run-pass liveintervals,twoaddressinstruction -verify-machineinstrs %s -o -
```
Maybe also remove `-verify-machineinstrs`.

Reply:
llc has bad defaults and doesn't run the verifier after mir tests